superbench/common/utils/file_handler.py

```diff
@@ -57,16 +57,26 @@ def create_sb_output_dir(output_dir=None):
 def get_sb_config(config_file):
     """Read SuperBench config yaml.
 
-    Read config file, use default config if None is provided.
+    Read config file, detect Azure SKU and use corresponding config if None is provided.
 
     Args:
         config_file (str): config file path.
 
     Returns:
         OmegaConf: Config object, None if file does not exist.
     """
-    default_config_file = Path(__file__).parent / '../../config/default.yaml'
-    p = Path(config_file) if config_file else default_config_file
+    p = Path(str(config_file))
+    if not config_file:
+        config_path = Path(__file__).parent / '../../config'
+        p = config_path / 'default.yaml'
+        vm_size = get_vm_size().replace('Standard_', '').lower()
```

Comment:
Maybe we don't need to replace?

Reply:
the prefix is duplicate in all skus, and current [config files](https://github.com/microsoft/superbenchmark/tree/main/superbench/config/azure/inference) do not include it

Reply:
Yes, we should replace all current config name.

Reply:
updated to keep "standard_" prefix